sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -194,4 +195,22 @@ public IdentityClientOptions setHttpClient(HttpClient httpClient) {
         this.httpClient = httpClient;
         return this;
     }
+
+    /**
+     * Specifies the database to extract intellij cached credentials from.
+     * @param keepPassDatabasePath the database to extract intellij credentials from.
+     * @return IdentityClientOptions
+     */
+    public IdentityClientOptions setKeepPassDatabasePath(String keepPassDatabasePath) {
+        this.keepPassDatabasePath = keepPassDatabasePath;
+        return this;
```

Comment:
Why are we adding this to the IdentityClientOptions? isn't this the base options for all credentials? Are users ever expected to create the IdentityClientOptions themselves?

Reply:
My bad I didn't see it was in implementation